frontend/src/metabase/query_builder/components/view/sidebars/ChartTypeSidebar.tsx

```diff
@@ -101,10 +100,9 @@ const ChartTypeSidebar = ({
         openChartSettings(e);
       } else {
         const newQuestion = question.setDisplay(display).lockDisplay(); // prevent viz auto-selection
-
         updateQuestion(newQuestion, {
-          reload: false,
```

Comment:
Hey Jesse, Apologies for the delayed response.

I removed reload here because the updateQuestion (in `updateQuestion.ts`) function doesn't seem to use it as an input. I likely didn't realize that when I was working on this last time, I only noticed when I imported the options.